context/app/static/js/components/detailPage/provenance/ProvVis/__tests__/ProvVis.spec.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { render, unmountComponentAtNode } from 'react-dom';
+// eslint-disable-next-line import/no-unresolved
+import { render } from 'test-utils/functions';
```

Comment:
Again with module name mapper, this shouldn't be necessary? Are the aliases not working?

Reply:
To be reverted 👍